backend/mora/service/detail_reading.py

```diff
@@ -655,6 +662,7 @@ def as_values(vs):
             **address.get_one_address(funcobj),
         }
         for funcid, funcobj in address_functions.items()
+        if not is_empty_lora_object(funcobj)
```

Comment:
Perhaps:

```python
if util.is_reg_valid(funcobj)
```

I believe this is consistent with the other reading code and arguably a more thorough check.